pom.xml

```diff
@@ -1588,34 +1588,43 @@
-                                         <bundledSignature>jdk-deprecated</bundledSignature>
-                                         <bundledSignature>jdk-system-out</bundledSignature>
-                                         <bundledSignature>jdk-non-portable</bundledSignature>
-                                         <bundledSignature>jdk-internal</bundledSignature>
-                                     </bundledSignatures>
-                                     <failOnMissingClasses>false</failOnMissingClasses>
-                                     <failOnUnresolvableSignatures>false</failOnUnresolvableSignatures>
-                                     <!--
-                                     The method SharedSessionContract#getTransaction() should never be used
-                                     as it is only reliable in certain configurations.
-                                      -->
-                                     <signatures>
+                            <configuration>
+                                 <targetVersion>10</targetVersion>
+                                 <!-- if the used Java version is too new, don't fail, just do nothing: -->
+                                 <failOnUnsupportedJava>false</failOnUnsupportedJava>
+                                 <excludes>
+                                     <!-- JBoss Logger formats messages according to default Locale (bug or feature?) -->
+                                     <exclude>**/*_$logger.class</exclude>
+                                     <!-- JBoss message bundles format messages -->
+                                     <exclude>**/*_$bundle.class</exclude>
+                                 </excludes>
+                                 <bundledSignatures>
+                                     <!-- This will automatically choose the right signatures based on 'targetVersion': -->
+                                     <bundledSignature>jdk-unsafe</bundledSignature>
+                                     <bundledSignature>jdk-deprecated</bundledSignature>
+                                     <bundledSignature>jdk-system-out</bundledSignature>
+                                     <bundledSignature>jdk-non-portable</bundledSignature>
+                                     <bundledSignature>jdk-internal</bundledSignature>
+                                 </bundledSignatures>
+                                 <failOnMissingClasses>false</failOnMissingClasses>
+                                 <failOnUnresolvableSignatures>false</failOnUnresolvableSignatures>
+                                 <!--
+                                 The method SharedSessionContract#getTransaction() should never be used
+                                 as it is only reliable in certain configurations.
+                                  -->
+                                 <signatures>
 org.hibernate.SharedSessionContract#getTransaction()
```

Comment:
Is there a specific reason not to include this in `forbidden-runtime.txt` and `forbidden-tests.txt`?

Reply:
I could move it. The only reason I didn't was that I prefer small, fine-grained patches :)